src/components/kids/index.vue

```diff
@@ -67,6 +67,18 @@ export default {
         })
         this.result = val
       }
+    },
+    baby () {
+      var sef = this
+      this.select = 'Dress'
+      if (this.select) {
```

Comment:
if (this.select) {} <- ဒါသံုးတာ ဘာအတြက္လဲ ညီမ
မသံုးလို႔ေရာ မရဘူးလား
ျပန္းစစ္ေပးပါ